components/Navigation.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  Box,
+  BottomNavigation,
+  BottomNavigationAction,
+  Paper,
+} from "@mui/material";
+import HomeIcon from "@mui/icons-material/Home";
+import SearchIcon from "@mui/icons-material/Search";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import AddIcon from "@mui/icons-material/Add";
+import AccountBoxIcon from "@mui/icons-material/AccountBox";
+import Link from "next/link";
+
+import { useState } from "react";
+export default function Navigation() {
+  const [value, setValue] = useState(0);
+  return (
+    <Box sx={{ width: 500 }}>
```

Comment:
What is this `<Box>` element for? I don't see a difference when I remove it.

Reply:
I removed it.